### PR TITLE
Add descriptions for undocumented colorpicker properties

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

File: src/app/public/modules/colorpicker/types/colorpicker-axis.ts [hunk 1]
```diff
@@ -1,8 +1,26 @@
 // spell-checker:ignore Colorpicker,
+/**
+ * Specifies the
+ */
 export interface SkyColorpickerChangeAxis {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/colorpicker/types/colorpicker-axis.ts:5]: I'm not sure why this property is part of the public API... its only used in the `SkyColorpickerSliderDirective` (which is internal), and as a setter in the colorpicker component itself - but not an`@Input()` setter. @Blackbaud-SteveBrush could you shed some light on this? Do we need to document?

File: src/app/public/modules/colorpicker/types/colorpicker-color.ts [hunk 2]
```diff
@@ -1,6 +1,18 @@
 // spell-checker:ignore Colorpicker,
+/**
+ * Specifies the
+ */
 export interface SkyColorpickerChangeColor {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/colorpicker/types/colorpicker-color.ts:5]: Same question as above - re: Public API?

File: src/app/public/modules/colorpicker/types/colorpicker-result.ts [hunk 3]
```diff
@@ -1,4 +1,10 @@
 import { SkyColorpickerOutput } from './colorpicker-output';
- export interface SkyColorpickerResult {
+/**
+ * Specifies the selected color.
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/colorpicker/types/colorpicker-result.ts:3]: This type is used when the users **apply** a color. Should we change this to say something to match the description you used in the `SkyColorpickerComponent` ?

File: src/app/public/modules/colorpicker/types/colorpicker-result.ts [hunk 4]
```diff
@@ -1,4 +1,10 @@
...
+ */
+export interface SkyColorpickerResult {
+  /**
+   * Emits an object that describes the color that users select in the colorpicker.
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/colorpicker/types/colorpicker-result.ts:7]: This is a type and not an emitter. Should it say something like...
```
Describes the color that...
```

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
A few questions...

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-flesh-out-jsdocs-093020](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-flesh-out-jsdocs-093020)

File: src/app/public/modules/colorpicker/colorpicker-classes.ts [hunk 5]
```diff
@@ -2,15 +2,39 @@
 
 export class SliderPosition {
```
[the remark this answers is not in the document]
Comment (blackbaud-johnly) [reply src/app/public/modules/colorpicker/colorpicker-classes.ts:3]: D'oh! We talked about that, and I got my wires crossed. I'll add `@internal`.

File: src/app/public/modules/colorpicker/types/colorpicker-axis.ts (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/public/modules/colorpicker/types/colorpicker-axis.ts:5]: Not sure if Steve is around today. It sounds like it shouldn't be in the docs, so I'm going to go ahead and add `@internal`. But I'll leave this comment thread open to make sure we confirm with Steve before merging.

File: src/app/public/modules/colorpicker/types/colorpicker-result.ts (hunk 3 above)
Comment (blackbaud-johnly) [reply src/app/public/modules/colorpicker/types/colorpicker-result.ts:3]: How about `Indicates the color that users apply when they select Apply in the colorpicker.`

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-flesh-out-jsdocs-093020](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-flesh-out-jsdocs-093020)

File: src/app/public/modules/colorpicker/types/colorpicker-result.ts (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/colorpicker/types/colorpicker-result.ts]: Works for me!

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-flesh-out-jsdocs-093020](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-flesh-out-jsdocs-093020)

File: src/app/public/modules/colorpicker/types/colorpicker-axis.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/colorpicker/types/colorpicker-axis.ts:5]: Yep, internalize it.

File: src/app/public/modules/colorpicker/types/colorpicker-color.ts (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/colorpicker/types/colorpicker-color.ts:5]: Sounds good to me.

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**